text.txt

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item"><a class="nav-link" href="#">Portfolio</a></li>
+                <li class="nav-item"><a class="nav-link" href="#">About</a></li>
+                <li class="nav-item"><a class="nav-link" href="#">Contact</a></li>
+            </ul>
+            <!-- <button><img src="assets/images/menu.svg" alt="Menu"></button> -->
+        </nav>
+    </header>
+    <main>
+        <section>
+                <h2>Hey there.
+                    I’m Daniel. <br>
+                    <span>I’m a software developer</span>
+                </h2>     
+                <div>
+                    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                </div>
+                <ul>
+                    <li><img src="assets/images/twitter.svg" alt="twitter logo"></li>
+                    <li><img src="assets/images/linkedin.svg" alt="linkedin logo"></li>
+                    <li><img src="assets/images/medium.svg" alt="medium logo"></li>
+                    <li><img src="assets/images/github.svg" alt="github logo"></li>
+                    <li><img src="assets/images/angellist.svg" alt="angellist logo"></li>
+                </ul>
+                <div>
+                    <img src="assets/images/normalbottun.svg" alt="normalbottun">
+                </div>
+            </section>
+        <section>
+            <div>
+                <h2>My Recent Works</h2>
+                <img src="assets/images/indicator.svg" alt="">
+            </div>
+            <div>
+                <div>
+                    <div>
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                        <ul>
```

Comment:
Please use the proper semantic tags. I realized you used h1 for a subheading, you can use other tags instead for a subheading. so you can have a well-structured code. follow the instruction.